bundles/org.openhab.binding.siemenshvac/src/main/java/org/openhab/binding/siemenshvac/internal/metadata/SiemensHvacMetadataRegistryImpl.java

```diff
@@ -0,0 +1,1270 @@
+                                    idxCell++;
+                                }
+
+                                if ("".equals(userName)) {
+                                    continue;
+                                }
+
+                                Pattern pattern4 = Pattern.compile("userid=(.+?)");
+                                Matcher matcher4 = pattern4.matcher(userEdit);
+
+                                SiemensHvacMetadataUser user = new SiemensHvacMetadataUser();
+                                user.setName(userName);
+
+                                if (matcher4.find()) {
+                                    userId = matcher4.group(1);
+                                    user.setId(Integer.parseInt(userId));
+                                } else {
+                                    userId = null;
+                                    user.setId(-1);
+                                }
+
+                                request = "main.app?section=settings&subsection=user&action=modify";
+                                if (userId != null) {
+                                    request = request + "&userid=" + userId;
+                                }
+                                response = lcHvacConnector.doBasicRequest(request);
+
+                                Pattern pattern5 = Pattern.compile(
+                                        "<select name=\\\"language\\\".*>((.*|\\n)*?)</select>", Pattern.MULTILINE);
+                                Matcher matcher5 = pattern5.matcher(response);
+
+                                if (matcher5.find()) {
+                                    String optionsList = matcher5.group(1);
+
+                                    Pattern pattern6 = java.util.regex.Pattern.compile(
+                                            "<option value=\\\"([^ ]*)\\\"(.*)>(.*)</option>", Pattern.MULTILINE);
+                                    Matcher matcher6 = pattern6.matcher(optionsList);
+
+                                    while (matcher6.find()) {
+                                        String id = matcher6.group(1);
+                                        String opt = matcher6.group(2);
+                                        String lang = matcher6.group(3);
+
+                                        if (opt.indexOf("selected") >= 0) {
+                                            user.setLanguage(lang);
+                                            user.setLanguageId(Integer.parseInt(id));
+                                        }
+                                    }
+                                }
+
+                                userList.put(userName, user);
+                            }
+
+                            idx++;
+
+                        }
+                    }
+                }
+            }
+        } catch (Exception e) {
+            logger.error("siemensHvac:ResolveDpt:Error during reading user info: {}", e.getLocalizedMessage());
```

Comment:
Yes, I remove the logging.
If there is an error there in this initialization phase, binding will go offline mode because the exception is catch from initializeCode()

 private void initializeCode() {
        try {
            metaDataRegistry.readMeta();
            updateStatus(ThingStatus.ONLINE);
        } catch (SiemensHvacException ex) {
            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
                    String.format("@text/thing-handler.msg.errorGatewayInit", ex.getMessage()));
        }
    }